.gitignore

```diff
@@ -9,3 +9,49 @@ aiohttp
 /hangupsbot/__pycache__
 /build
 /scripts
+.idea
+
+# telesync could keep photos, and probably we don't want that photos on git repo
+*.jpg 
+
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
```

Comment:
I just wanted to keep my IDE configuration to myself. It shouldn't be required for bot functionality etc but I am a lazy guy to learn how to exclude files from pull requests